src/cpp/shared_core/include/shared_core/FilePath.hpp

```diff
@@ -573,6 +573,14 @@ class FilePath
     */
    Error openForWrite(std::shared_ptr<std::ostream>& out_stream, bool in_truncate = true) const;
 
+   /*
```

Comment:
Nit: methods in shared_core files are alphabetical. Could you move this just before the `private:` section? Also in the CPP file.

Reply:
Done